.evergreen/config.yml

```diff
@@ -203,6 +203,19 @@ functions:
         content_type: application/x-gzip
         display_name: "fuzz.tgz"
 
+  upload-raw-test-suite:
```

Comment:
IIUC the reason this function is not in the `upload-mo-artifacts`'s command array is that this needs to be used in multiple tasks. Is it possible that those cases result in workflows that do not call the [post](https://github.com/blink1073/mongo-go-driver/blame/GODRIVER-2960/.evergreen/config.yml#L1021-L1021C5) functions?

Reply:
Good point, I grouped them together and renamed it to `handle-test-artifacts`, since we're already uploading the fuzz file there.